src/components/Button/ButtonConstants.ts

```diff
@@ -0,0 +1,34 @@
+import { SIZES } from "../../constants/sizes";
+
+export enum ButtonType {
+  PRIMARY = "primary",
+  SECONDARY = "secondary",
+  TERTIARY = "tertiary"
+}
+
+export enum ButtonColor {
+  PRIMARY = "primary",
+  POSITIVE = "positive",
+  NEGATIVE = "negative",
+  ON_PRIMARY_COLOR = "on-primary-color",
+  ON_INVERTED_BACKGROUND = "on-inverted-background"
+}
+
+const OLD_BUTTON_SIZES: Record<string, string> = {
+  sm: SIZES.SMALL,
+  md: SIZES.MEDIUM,
+  lg: SIZES.LARGE
+};
```

Comment:
I think `as const` will give us better typing in this case
```suggestion
const OLD_BUTTON_SIZES = {
  sm: SIZES.SMALL,
  md: SIZES.MEDIUM,
  lg: SIZES.LARGE
} as const;
```